constants/currencies.ts

```diff
@@ -401,6 +401,16 @@ const SolanaDevnet: CurrencyMetadata[] = [
   },
 ];
 
+// Question: what date should here?
+// Temporary fake data for creating a build
+const FakeData: CurrencyMetadata[] = [
+  {
+    address: "LOCALHOST_TOKEN",
+    name: "localhost",
+    symbol: "ETH",
+  },
+];
+
```

Comment:
@jnsdls Need your input here. What should be the metadata for localhost and hardhat, or should they be removed from the CURRENCIES object?

Reply:
This should come from the chain config not from a constants map

Reply:
With that said I think we do need to add in currencies that we know for our default chains, so really this becomes a job of merging those for chains we know special currency addresses for

Reply:
That should probably happen still in the chains package though…